bahmni-mart/dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM openjdk:8-jre-alpine
```

Comment:
In the rest of the project, Dockerfiles are named `Dockerfile` (capital D). Unless you think we should change this, I suggest you name it this way too.

Reply:
Sure would change the name.